From 584a82ea2033301949e002b18cfc59494488f210 Mon Sep 17 00:00:00 2001
From: Zhendong Li <54206290+DMax1314@users.noreply.github.com>
Date: Mon, 23 Feb 2026 01:59:43 -0500
Subject: [PATCH 2679/5749] Update Kimi Code Subscription link in README

The link does not work anymore. You can use your referral link if you'd like. This one I'm sharing is just a direct link.
---
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.md b/README.md
index 1ce701e7..00bde1db 100644
--- a/README.md
+++ b/README.md
@@ -134,7 +134,7 @@ Everything below, every feature, every optimization, you don't need to know it.
 
 Even only with following subscriptions, ultrawork will work well (this project is not affiliated, this is just personal recommendation):
 - [ChatGPT Subscription ($20)](https://chatgpt.com/)
-- [Kimi Code Subscription ($0.99) (*only this month)](https://www.kimi.com/membership/pricing?track_id=5cdeca93-66f0-4d35-aabb-b6df8fcea328)
+- [Kimi Code Subscription ($0.99) (*only this month)](https://www.kimi.com/kimiplus/sale)
 - [GLM Coding Plan ($10)](https://z.ai/subscribe)
 - If you are eligible for pay-per-token, using kimi and gemini models won't cost you that much.
 
-- 
2.39.2

